portalwire/portal_protocol.go

```diff
@@ -249,7 +253,8 @@ type PortalProtocol struct {
 	portalMetrics  *portalMetrics
 	PingExtensions pingext.PingExtension
 
-	disableTableInitCheck bool
+	disableTableInitCheck  bool
+	portalProtocolVersions protocolVersions
```

Comment:
in this struct, we could remove prefix such 'portal' and 'portalProtocol' for field names, better to change other field names together